src/scripts/handleChat.js

```diff
@@ -147,14 +165,27 @@ ComfyJS.onChat = function(user, messageContents, flags, self, extra) {
 ComfyJS.onMessageDeleted = function(id, extra) {
 	const messageToDelete = document.querySelector(`[data-twitch-message="${id}"]`);
 	if (messageToDelete) {
-		let wasFirstInGroup = messageToDelete.getAttribute('data-twitch-first-message-in-group');
-		let group = messageToDelete.getAttribute('data-twitch-message-group');
-		let hasNextInGroup = messageToDelete.nextSibling && messageToDelete.nextSibling.getAttribute('data-twitch-message-group') === group;
-		if (wasFirstInGroup && hasNextInGroup) {
-			messageToDelete.nextSibling.setAttribute('data-twitch-first-message-in-group', true);
-		}
-		messageToDelete.remove();
+		removeMessageFromDomAndShiftOthers(messageToDelete);
 	}
 }
 
-ComfyJS.Init(null, null, watchedChannels.split(' '));
\ No newline at end of file
+function removeMessageFromDomAndShiftOthers(messageToDelete) {
+	let wasFirstInGroup = messageToDelete.getAttribute('data-twitch-first-message-in-group');
+	let group = messageToDelete.getAttribute('data-twitch-message-group');
+	let hasNextInGroup = messageToDelete.nextSibling && messageToDelete.nextSibling.getAttribute('data-twitch-message-group') === group;
+	if (wasFirstInGroup && hasNextInGroup) {
+		messageToDelete.nextSibling.setAttribute('data-twitch-first-message-in-group', true);
+	}
+	messageToDelete.remove();
+}
+
+
+let testDict
```

Comment:
Do we still need this variable?

Reply:
oh no i have brought great shame to my bloodline. we certainly do not and that shouldn't have been pushed in the first place 🤐